### PR TITLE
added link to mail and link to tel

### PR DESCRIPTION
**Pull request by annaeklov**
image

### CODE REVIEW

**Comment by rubikuo** [comment]
Looks good!

File: src/pages/ItemDetails.tsx [hunk 1]
```diff
@@ -469,6 +475,9 @@ const ItemDetails: FC<ParamTypes> = () => {
     history.goBack();
   };
 
+  let mailtoHref = `mailto:${item.email}?subject=Email från Haffa`;
```
Comment (jonatanhanson) [inline src/pages/ItemDetails.tsx:478]: change let to const

File: src/pages/ItemDetails.tsx [hunk 2]
```diff
@@ -469,6 +475,9 @@ const ItemDetails: FC<ParamTypes> = () => {
...
+  let telHref = `tel:${item.phoneNumber}`;
```
Comment (jonatanhanson) [inline src/pages/ItemDetails.tsx:479]: change let to const

**Review by jonatanhanson — APPROVED** [review]
Works as intended 👍 
Added comments for a small change.

File: src/pages/ItemDetails.tsx [hunk 3]
```diff
@@ -689,11 +696,11 @@ const ItemDetails: FC<ParamTypes> = () => {
             </div>
             <div className="contactInfo">
               <MdPhone />
-              <p>{item.phoneNumber}</p>
```
Comment (jonatanhanson) [inline src/pages/ItemDetails.tsx:692]: Hide the whole phone-field if phonenumber does not exist, since its not a required field.

**Review by JohannaENilsson — APPROVED** [review]
Looks great.

**Pull request merged**